msteams-platform/sbs-teams-app-test-tool.yml

```diff
@@ -0,0 +1,351 @@
+### YamlMime:Tutorial
+title: Debug your bot using Teams App Test Tool
+metadata:
+  title: Debug your bot using Teams App Test Tool
+  description: With this learning module, you learn how to emulate the Teams experience for your bot in Teams App Test Tool.
+  audience: Developer
+  level: Advanced
+  ms.date: 10/21/2021
+  ms.topic: interactive-tutorial
+  nextTutorialHref: toolkit/debug-your-teams-app-test-tool.md
+  nextTutorialTitle: Teams App Test Tool
+  ms.custom: mvc
+  ms.localizationpriority: high
+items:
+- durationInMinutes: 1
+  content: |
+    Start Microsoft Teams app development with your Teams AI chat bot app and debug with the Teams App Test Tool (Test Tool). Test Tool makes debugging bot-based apps effortless. You can chat with your bot and see its messages and Adaptive Cards as they appear in Teams. You don't need a Microsoft 365 developer account, tunneling, or Teams app and bot registration to use Test Tool. You can also mock an activity in Test Tool using activity triggers.
+
+    > [!NOTE]
+    >
+    > * Teams App Test Tool is available only in the latest prerelease version of the Teams Toolkit.
+    > * Teams App Test Tool is supported only for desktop and web clients.
+
+    This step-by-step guide helps you to build AI chat bot using Teams Toolkit and debug AI chat bot with the Test Tool. You'll see the following output after you've completed this guide, where the user can access and use the AI chat bot:
+
+    :::image type="content" source="~/assets/images/teams-toolkit-v2/debug/test-tool.png" lightbox="~/assets/images/teams-toolkit-v2/debug/test-tool.png" alt-text="Screenshot shows the bot open in Test Tool.":::
```

Comment:
Would you please replace with the following snapshot? Because the latest version of Teams App Test Tool already supported typing activity. The warnings output from earlier version are no longer exist anymore.

![image](https://github.com/MicrosoftDocs/msteams-docs/assets/15644078/e77c9612-abe1-436d-aa78-5daafcec0ec7)

Reply:
Updated the screenshot.